# HG changeset patch
# User Luca Greco <luca.greco@alcacoop.it>

TMP: workaround on devtools/shared/webconsole/utils

---
 devtools/shared/webconsole/utils.js | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/devtools/shared/webconsole/utils.js b/devtools/shared/webconsole/utils.js
index 45dc8be..6897461 100644
--- a/devtools/shared/webconsole/utils.js
+++ b/devtools/shared/webconsole/utils.js
@@ -1187,16 +1187,19 @@ WebConsoleCommands._registerOriginal("$$", function JSTH_$$(aOwner, aSelector)
 /**
  * Returns the result of the last console input evaluation
  *
  * @return object|undefined
  * Returns last console evaluation or undefined
  */
 WebConsoleCommands._registerOriginal("$_", {
   get: function(aOwner) {
+    if (!aOwner.consoleActor) {
+      return null;
+    }
     return aOwner.consoleActor.getLastConsoleInputEvaluation();
   }
 });
 
 
 /**
  * Runs an xPath query and returns all matched nodes.
  *

